### PR TITLE
Adding font-stack helpers

### PR DESCRIPTION
**Pull request by ollips**
Task: task-419077

Link: preview-135

Creating helper classes `font-family-monospace` and `font-family-quote`.

## Testing

1. Open any page. Inspect any text paragraph/title. Try to add either of the newly created classes. You should see how the font family changes accordingly to the applied class.

### CODE REVIEW

File: css/src/atomics/typography.scss [hunk 1]
```diff
@@ -1,3 +1,15 @@
+// Font stack
+
+.is-monospace {
```
Comment (wibjorn) [inline css/src/atomics/typography.scss:3]: These should match the same pattern as other atomics.

```
.font-family-monospace
.font-family-quote
```

File: css/src/atomics/typography.scss [hunk 2]
```diff
@@ -1,3 +1,15 @@
...
+	font-family: $monospace-font-stack !important;
+	-moz-osx-font-smoothing: auto;
+	-webkit-font-smoothing: auto;
+}
+
+.is-quote {
```
Comment (wibjorn) [inline css/src/atomics/typography.scss:9]: This item, at present at least, has very focused usefulness. I think we can remove it from the framework.

Comment (wibjorn) [inline css/src/atomics/typography.scss:9]: `.font-family-quote`

File: css/src/atomics/typography.scss (hunk 1 above)
Comment (ollips) [reply css/src/atomics/typography.scss]: updated

File: css/src/atomics/typography.scss (hunk 2 above)
Comment (ollips) [reply css/src/atomics/typography.scss]: updated

**Comment by wibjorn** [comment]
Does this need documentation?

**Review by wibjorn — APPROVED** [review]
Looks good, but do we need documentation for this?

**Comment by ollips** [comment]
> Looks good, but do we need documentation for this?

Gonna be covered in a separate task ;)

**Pull request merged**